### ISSUE TITLE
Proposal: Add more examples using cpp-sdk

### ISSUE THREAD

**Issue by Xunzhuo** [proxy-wasm/proxy-wasm-cpp-sdk#104]
I think the official example just shows very basic use of cpp-sdk .  Can we add more extensions built by wasm using cpp-sdk ?

**Comment by PiotrSikora** [comment]
Yes, ideally, we should have the same set of examples implemented in all Proxy-Wasm SDKs (e.g. see [examples in Proxy-Wasm Rust SDK](https://github.com/proxy-wasm/proxy-wasm-rust-sdk/tree/master/examples)).

PRs welcome!

**Comment by Xunzhuo** [comment]
Sure, I am working on this recently~ If there is some good examples, I will give PRs~ @PiotrSikora
